Python/Product/PythonTools/PythonTools/Repl/SelectableReplEvaluator.cs

```diff
@@ -201,11 +202,12 @@ string windowId
             await eval.InitializeAsync();
 
             var view = eval?.CurrentWindow?.TextView;
-            var buffer = eval?.CurrentWindow?.CurrentLanguageBuffer;
-            if (view != null && buffer != null) {
+            var buffer = PythonTextBufferInfo.TryGetForBuffer(eval?.CurrentWindow?.CurrentLanguageBuffer);
+            var pyEval = eval as IPythonInteractiveIntellisense;
+            if (view != null && buffer != null && pyEval != null) {
                 var controller = IntellisenseControllerProvider.GetOrCreateController(_serviceProvider, _serviceProvider.GetComponentModel(), view);
-                controller.DisconnectSubjectBuffer(buffer);
-                controller.ConnectSubjectBuffer(buffer);
+                var entry = pyEval.Analyzer.GetAnalysisEntryFromPath(pyEval.AnalysisFilename);
+                Debug.Assert(entry == buffer.TrySetAnalysisEntry(entry, buffer.AnalysisEntry));
```

Comment:
Do you want TrySetAnalysisEntry to run in release builds?
https://stackoverflow.com/questions/13383651/will-the-expression-provided-to-debug-assert-be-evaluated-in-a-release-build

Reply:
Um... yes... my bad

Reply:
In fact, that whole function should be simplified, because I moved the Intellisense handling into the Initialize call. I missed this when I went looking for dead code (though I'll make sure it's actually dead...)